2018/2018_File_Generation_Features.md

```diff
@@ -0,0 +1,67 @@
+# 2018 File Generation Features
+
+## The FileGenerator Class. 
+The `FileGenerator()` class is located in [`file_generator.py`](https://github.com/cfpb/hmda-test-files/blob/master/2018/python/file_generator.py) and instantiates without any inputs in the function call. The class uses data stored with yaml configuration files in the 
+[`python/configurations/`](https://github.com/cfpb/hmda-test-files/tree/master/2018/python/configurations) directory. The following yaml configuration files can be altered to create HMDA submission test files.  
+
+-[`test_filepaths.yaml`](https://github.com/cfpb/hmda-test-files/blob/master/2018/python/configurations/test_filepaths.yaml) holds the filepath names for storing clean files, edit test files, log files, and edit reports as well as the paths to the TS and LAR schemas. 
```

Comment:
Change `and edit reports as well as the paths to the TS and LAR schemas. ` to `,edit reports and the paths to the TS and LAR schemas. `

Reply:
For each of these configuration sections, can you please add a bullet list (not numbered) that contains each of the variables in the yaml file? The goal here is to show the user what they can change without forcing them into the yaml file. The bulleted list is a better visual cue than using a sentence structure.

Reply:
Sure! I also like having all the configurations in one place.

Reply:
I am actually putting in a table for these. This should render neatly, and save some space for the user.